docs/resources/logdna_alert.md

```diff
@@ -75,45 +75,45 @@ resource "logdna_alert" "my_alert" {
 
 ## Argument Reference
 
-The following arguments are supported:
+The following arguments are supported by `logdna_alert`:
 
 - `name`: (Required) The name this Preset Alert will be given, type _string_
 
 ### email_channel
 
 `email_channel` supports the following arguments:
 
-- `emails`: _(Required)_ An array of email addresses (each email is of type _string_) to notify in the Alert
-- `immediate`: _(Optional)_ Whether the Alert will trigger immediately after the trigger limit is reached. _Note: Immediate can only be set to `true` for presence alerts_. Valid options are `true` and `false` for presence Alerts and `false` for absence Alerts, type _string_ (**Default: "false"**)
-- `operator`: _(Optional)_ Whether the Alert will trigger on the presence or absence of logs. Valid options are `presence` and `absence`, type _string_ (**Default: "presence"**)
-- `terminal`: _(Optional)_ Whether the Alert will trigger after the `triggerinterval` if the Alert condition is met (e.g., send an Alert after 30s). Valid options are `true` and `false` for presence Alerts and `true` for absence Alerts, type _string_ (**Default: "true"**)
-- `timezone`: _(Optional)_ Which time zone the log timestamps will be formatted in. Timezones are represented as [database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), type _string_
-- `triggerinterval`: _(Optional)_ Interval which the Alert will be looking for presence or absence of log lines. For presence Alerts, valid options are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. For absence Alerts, valid options are: `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. Type _string_ (**Defaults: "30" for presence; "15m" for absence**)
-- `triggerlimit`: _(Required)_ Number of lines before the Alert is triggered. (eg. Setting a value of `10` for an `absence` Alert would alert you if `10` lines were not seen in the `triggerinterval`), type _integer_
+- `emails`: **_[]string (Required)_** An array of email addresses (strings) to notify in the Alert
+- `immediate`: **_string_** _(Optional; Default: `"false"`)_ Whether the Alert will trigger immediately after the trigger limit is reached. _Note: Immediate can only be set to `"true"` for presence alerts_. Valid options are `"true"` and `"false"` for presence Alerts and `"false"` for absence Alerts.
```

Comment:
ugh, this has to match with the API, and I'm not sure of the rules there.  It's also worth noting that I just hit the problem where one of `immediate` or `terminal` has to be `true` (at least for webhooks), but I see no way to validate them both ahead of time.

We have to check the API functionality before this can be confirmed.

Reply:
Yeah makes sense, this might just be something we'll have to flag for now and come back to later once we know more

Reply:
Ok, it's a wording thing.  It tries to say that `true` AND `false` are valid options for `presence`, but only `false` is available for absence alerts.  Removing that first part should clear it up.

Reply:
fixed, good catch